app/lib/features/home/pages/home_shell.dart

```diff
@@ -140,8 +141,8 @@ class _HomeShellState extends ConsumerState<HomeShell> {
     }
 
     final bottomBarNav = ref.watch(bottomBarNavProvider(context));
-    final bottomBarIdx =
-        ref.watch(currentSelectedBottomBarIndexProvider(context));
+    // final bottomBarIdx =
```

Comment:
Why is this commented out? We don't need this index tracking anymore?

Reply:
I have kept that on commented stage for the reference if we need at any point of time.
But ya we don't need that index tracking anymore as we are using index of `navigationShell`. :)

Reply:
@kumarpalsinh25 remove it. for "reference" we always have the git history :) .